test/powershell/Language/Scripting/TryCatch.Tests.ps1

```diff
@@ -0,0 +1,651 @@
+
+#############################################################
+#
+# Test simple parsing, ensure newlines allowed everywhere
+#
+#############################################################
+
+Describe "Test try/catch" -Tags "CI" {
+
+    BeforeAll {
+        function AssertArraysEqual ($result, $expected)
+        {
+            $result.Count | Should Be $expected.Count
+            for ($i = 0; $i -lt $result.Count; $i++) {
+                $result[$i] | Should Be $expected[$i]
+            }
+        }
+    }
+    
+    It "Test simple parsing, ensure newlines allowed everywhere" {
+        {
+            try
+            {
+            }
+            catch
+            {
+            }
+
+            try
+            {
+            }
+            catch
+            [int]
+            {
+            }
+
+            try
+            {
+            }
+            catch
+            [int]
+            ,
+            [char]
+            {
+            }
+
+            try
+            {
+            }
+            finally
+            {
+            }
+
+            try
+            {
+            }
+            catch
+            {
+            }
+            finally
+            {
+            }
+
+            try
+            {
+            }
+            catch
+            [int]
+            {
+            }
+            finally
+            {
+            }
+
+            try
+            {
+            }
+            catch
+            [int]
+            ,
+            [char]
+            {
+            }
+            finally
+            {
+            }
+        } | Should Not Throw
```

Comment:
Right, if parsing failed, `| Should Not Throw` will never run. The original test was to make sure those use of try/catch would not cause parsing error.
How about I just have a dummy `$true | Should Be $true` following the try/catch blocks? So if no parsing error, the test will pass.